theme/static/style/style.css

```diff
@@ -405,6 +419,26 @@ pre code {
         text-align: left;
     }
 
+    /*Screenshots section.*/
+    .screenshots .col-2, .img-index {
+        display: none;
+        width: 0;
+    }
+
+    .screenshots .col-8 {
+        width: 100%;
+    }
+
+    .screenshots .img {
+        max-width: 100%;
+        /*height: 100%;
+        bottom: 0;*/
+    }
+
+    .img-wrapper {
+        /*padding: 0;*/
+        max-width: 100%;
```

Comment:
This is good, you also need `box-sizing: border-box` to make the image fit. Also `padding: 0` because the style above adds more padding.